apps/website/.vuepress/code/demos/checkbox/inline-css.html

```diff
@@ -2,15 +2,15 @@
   <label class="clr-control-label">Inline checkbox example</label>
   <div class="clr-control-container clr-control-inline">
     <div class="clr-checkbox-wrapper">
-      <input type="checkbox" id="inline-checkbox1" name="checkbox-full" value="option1" class="clr-checkbox" />
+      <input type="checkbox" id="inline-checkbox1" name="checkbox-inline-1" value="option1" class="clr-checkbox" />
       <label for="inline-checkbox1" class="clr-control-label">Option 1</label>
     </div>
     <div class="clr-checkbox-wrapper">
-      <input type="checkbox" id="inline-checkbox2" name="checkbox-full" value="option2" class="clr-checkbox" />
+      <input type="checkbox" id="inline-checkbox2" name="checkbox-inline-1" value="option2" class="clr-checkbox" />
```

Comment:
I thought we were getting rid of duplicate names for checkboxes?

Reply:
Yes, thank you :) Mechanical error - now we can't easily cherrypick between angular and v4 branches, so a lot is done by hand.

Reply:
Done!